### ISSUE TITLE
Python3.x install error

### ISSUE THREAD

**Issue by azu** [mooz/percol#16]
Hi, I really enjoy using your terminal tool.

Trying to install `percol` on python 3 fails:

``` sh
$ python --version
Python 3.3.2
```

`$ pip install percol` 

``` sh
Downloading/unpacking percol
  You are installing an externally hosted file. Future versions of pip will default to disallowing externally hosted files.
  You are installing a potentially insecure and unverifiable file. Future versions of pip will default to disallowing insecure files.
  Downloading master
  Running setup.py egg_info for package percol
    Traceback (most recent call last):
      File "<string>", line 16, in <module>
      File "/private/var/folders/yp/n_vh185d1z39k_xmp5wwwm1m0000gn/T/pip_build_azu/percol/setup.py", line 5, in <module>
        import percol
      File "./percol/__init__.py", line 40, in <module>
        import debug, action
    ImportError: No module named 'debug'
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 16, in <module>

  File "/private/var/folders/yp/n_vh185d1z39k_xmp5wwwm1m0000gn/T/pip_build_azu/percol/setup.py", line 5, in <module>

    import percol

  File "./percol/__init__.py", line 40, in <module>

    import debug, action

ImportError: No module named 'debug'

----------------------------------------
Cleaning up...
Command python setup.py egg_info failed with error code 1 in /private/var/folders/yp/n_vh185d1z39k_xmp5wwwm1m0000gn/T/pip_build_azu/percol
Storing complete log in /Users/azu/.pip/pip.log
```

**Comment by clach04** [comment]
Seems to be a none issue with Python 3.5 and 3.12. recommend closing. @azu is this still an issue?

**Comment by hartwork** [comment]
Error `ModuleNotFoundError: No module named 'debug'` was fixed in commit 10a13ee7edded150b311156b70a2471d329421b4 in 2014 and percol >=v0.0.4. Latest `master` at 4b28037e328da3d0fe8165c11b800cbaddcb525e does not have this problem. I agree with @clach04 to close as fixed.
